src/Symfony/Bundle/FrameworkBundle/composer.json

```
@@ -34,7 +34,7 @@
     },
     "require-dev": {
         "doctrine/annotations": "^1.10.4",
-        "doctrine/cache": "~1.0",
+        "doctrine/cache": "^1.11",
```

This can also allow 2.0 as long as you're not using an actual implementation:
```suggestion
        "doctrine/cache": "^1.11|^2.0",
```